.ci/scripts/go-plus/github-differ/generate_comment.sh

```diff
@@ -45,73 +45,63 @@ if ! git diff --exit-code origin/$OLD_BRANCH origin/$NEW_BRANCH; then
     SUMMARY=`git diff origin/$OLD_BRANCH origin/$NEW_BRANCH --shortstat`
     DIFFS="${DIFFS}${NEWLINE}Terraform GA: [Diff](https://github.com/modular-magician/terraform-provider-google/compare/$OLD_BRANCH..$NEW_BRANCH) ($SUMMARY)"
 fi
-TPG_LOCAL_PATH_OLD="${TPG_LOCAL_PATH}old"
-mkdir -p $TPG_LOCAL_PATH_OLD
-cp -r $TPG_LOCAL_PATH/. $TPG_LOCAL_PATH_OLD
+popd
 
-git checkout origin/$NEW_BRANCH
-update_package_name "github.com/hashicorp/terraform-provider-google" "google/provider/new"
+mkdir -p $TPGB_LOCAL_PATH
+git clone -b $NEW_BRANCH $TPGB_SCRATCH_PATH $TPGB_LOCAL_PATH
+pushd $TPGB_LOCAL_PATH
+git fetch origin $OLD_BRANCH
+if ! git diff --exit-code origin/$OLD_BRANCH origin/$NEW_BRANCH; then
+    SUMMARY=`git diff origin/$OLD_BRANCH origin/$NEW_BRANCH --shortstat`
+    DIFFS="${DIFFS}${NEWLINE}Terraform Beta: [Diff](https://github.com/modular-magician/terraform-provider-google-beta/compare/$OLD_BRANCH..$NEW_BRANCH) ($SUMMARY)"
+fi
 popd
 
 ## Breaking change setup and execution
-pushd $TPG_LOCAL_PATH_OLD
-git checkout origin/$OLD_BRANCH
-update_package_name "github.com/hashicorp/terraform-provider-google" "google/provider/old"
-popd
 set +e
 pushd $MM_LOCAL_PATH/tools/breaking-change-detector
-sed -i.bak -E "s~google/provider/(.*)/([0-9A-Za-z-]*)~google/provider/\1/google~" comparison.go
-go mod edit -replace google/provider/new=$(realpath $TPG_LOCAL_PATH)
-go mod edit -replace google/provider/old=$(realpath $TPG_LOCAL_PATH_OLD)
-go mod tidy
-export TPG_BREAKING="$(go run .)"
+cp -r $TPG_LOCAL_PATH old/
+cp -r $TPG_LOCAL_PATH new/
+make build OLD_REF=$OLD_BRANCH NEW_REF=$NEW_BRANCH
+export TPG_BREAKING="$(bin/breaking-change-detector)"
 retVal=$?
 if [ $retVal -ne 0 ]; then
     export TPG_BREAKING=""
     BREAKING_CHANGE_BUILD_FAILURE=$?
 fi
-set -e
-popd
 
-# TPGB
-mkdir -p $TPGB_LOCAL_PATH
-git clone -b $NEW_BRANCH $TPGB_SCRATCH_PATH $TPGB_LOCAL_PATH
-pushd $TPGB_LOCAL_PATH
-git fetch origin $OLD_BRANCH
-if ! git diff --exit-code origin/$OLD_BRANCH origin/$NEW_BRANCH; then
-    SUMMARY=`git diff origin/$OLD_BRANCH origin/$NEW_BRANCH --shortstat`
-    DIFFS="${DIFFS}${NEWLINE}Terraform Beta: [Diff](https://github.com/modular-magician/terraform-provider-google-beta/compare/$OLD_BRANCH..$NEW_BRANCH) ($SUMMARY)"
+rm -rf ./old/ ./new/
+cp -r $TPGB_LOCAL_PATH old/
+cp -r $TPGB_LOCAL_PATH new/
+make build OLD_REF=$OLD_BRANCH NEW_REF=$NEW_BRANCH
+export TPGB_BREAKING="$(bin/breaking-change-detector)"
+retVal=$?
+if [ $retVal -ne 0 ]; then
+    export TPGB_BREAKING=""
+    BREAKING_CHANGE_BUILD_FAILURE=$?
+fi
+BREAKINGCHANGES="$($script_dir/compare_breaking_changes.sh)"
+if [ $BREAKING_CHANGE_BUILD_FAILURE -ne 0 ]; then
+    BREAKINGCHANGES="## Breaking Change Detection Failed${NEWLINE}The breaking change detector crashed during execution. This is usually due to the downstream provider(s) failing to compile. Please investigate or follow up with your reviewer."
```

Comment:
```suggestion
    BREAKINGCHANGES="## Breaking Change Detection Failed${NEWLINE}The breaking change detector crashed during execution. This is usually due to the downstream provider(s) failing to compile. Please investigate or follow up with your reviewer.${NEWLINE}"
```

suggestion: add trailing newline. Not sure how the formatting will turn out otherwise. I would recommend breaking the provider downstream in the PR by changing a package name in mmv1 or something and seeing this workflow execute before checking in.

Reply:
I believe newlines after this text are already added at https://github.com/GoogleCloudPlatform/magic-modules/blob/cdabb6f304a7bc82131a39ca3d6a0b967161e4d5/.ci/scripts/go-plus/github-differ/generate_comment.sh#L156

Ack re: testing the workflow first :-)